example/cpp/unix/song.cpp

```diff
@@ -0,0 +1,121 @@
+/**
+ * 歌唱音声合成を行う。
+ *
+ * C APIの使い方であるため、書き方をCに寄せてある。
+ */
```

Comment:
これってコミッター側のメモですかね？
ユーザーに知ってほしいメッセージなんだとしたら伝えたいことが伝わる文章にすると良さそう。
メモならたぶんこうしておくと良さそう。

```suggestion
 */

// NOTE: C APIの使い方であるため、書き方をCに寄せてある。
```

Reply:
あ、後で微調整すればいいので、スルーしてマージでも大丈夫です！
ちなみにたぶんこういうことが言いたかったんじゃないかなと思ってます
```suggestion
 *
 * C言語としても参考できるよう、書き方をCに寄せてある。
 */
```

Reply:
なるほどです。

なぜCに寄せてるのか疑問を感じるはずだ、というのは僕も承知してました。
で、「意図的にCに寄せている」というのが「なぜ」の回答になっているという前提ですよね。
ここが僕と認識ずれてました。結局「なぜ？」ってなると思います。意図的に寄せてる理由がわからない。
元から書いてた「C APIの使い方であるため」も同様に「なぜ？」になる。

> ちなみに本当の理由としては、Discordの方でも発言したのですが、「[CXX](https://cxx.rs/)でのC++ APIを考えており、その場合既存のC++ exampleはC exampleとして退避させることになる。それを踏まえるとC++方式では書きたくないし、とはいえC++ APIも未定なので現行のexampleの形はなるだけそのままにしたい」というものです。
[追記] あ、C++を読み書きできる人に比べてCを読み書きできる人の方が多いだろうと踏んでるのも一つですね。

であれば、これからはそれを理由として書くのが良いと思います。
（これは「`C APIの使い方であるため`」ではない）

･･･････`C言語としても参考できるよう`が真芯を捉えてる気がしますね！
あるいは`// TODO: C++ APIができたらC++の書き方に変える`とか？